tpr/tpr.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 const (
-	tprInitRetries    = 30
-	tprInitRetryDelay = 3 * time.Second
+	tprInitMaxRetries    = 30
+	tprInitRetryInterval = 3 * time.Second
 )
 
 type Config struct {
```

Comment:
I thought it would be good to make the backoff configurable. At the moment clients rely on our default and have no chance to modify it to their needs.